### PR TITLE
Update release-team-leads slack group for 1.28 RT

### PR DESCRIPTION
**Pull request by marosset**
**Which issue(s) this PR fixes**:

Part of https://github.com/kubernetes/sig-release/issues/2223

/assign @gracenng @leonardpahlke

### CODE REVIEW

**Review by leonardpahlke — APPROVED** [review]
LGTM

File: communication/slack-config/sig-release/usergroups.yaml [hunk 1]
```diff
@@ -49,17 +49,17 @@ usergroups:
       - sig-release
     members:
       - cpanato # SIG Release Technical Lead
-      - Heba # 1.27 RT Lead Shadow
-      - hosseinsalahi # 1.27 RT Lead Shadow
-      - JamesLaverack # 1.27 Emeritus Advisor
+      - gracenng # 1.28 RT Lead
+      - helayoty # 1.28 RT Lead Shadow
+      - leonardpahlke # 1.28 Emeritus Adviser
```
Comment (palnabarun) [inline communication/slack-config/sig-release/usergroups.yaml:54]: small nit: this line should be added 2 lines below.

Comment (marosset) [reply communication/slack-config/sig-release/usergroups.yaml]: fixed - thanks!

**Comment by palnabarun** [comment]
Created https://github.com/kubernetes/test-infra/pull/29881 to resolve the job failure.

**Comment by marosset** [comment]
/test pull-community-tempelis-check
since https://github.com/kubernetes/test-infra/pull/29881 merged

**Comment by marosset** [comment]
> 2023/06/20 18:55:18 GET https://api.github.com/repos/kubernetes/kubernetes/releases: 403 API rate limit exceeded for 18.189.147.8. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) [rate reset in 12m25s]
exit status 1

:(

**Pull request merged**